ez_setup.py

```diff
@@ -1,284 +0,0 @@
-#!python
```

Comment:
Does deleting this file imply that users are restricted, now, to only Docker-based installation schemes?

Reply:
Nope. Docker was just an easy way to spin up Linux on OS X and make sure that pybert worked on both OS X and Linux.  Much quicker than a virtual machine.  Installation is still through the standard `pip install` or `conda install`